activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -491,7 +495,9 @@ def change_counter(key, amount, options)
                 c.expire(key, expires_in.to_i) if ttl < 0
               end
             else
-              count = c.incrby(key, amount)
+              count = c.pipelined do |pipeline|
+                pipeline.incrby(key, amount)
+              end.first
```

Comment:
This part of the change doesn't make sense.

Reply:
I agree. I'll revert this changes.